### PR TITLE
Add instructions for reverting from a split database to a single database

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) adds a new topic with instructions for reverting from a split database to a single database implementation.

See internal build `128/guides/v2.4/config-guide/revert-split-database.html`.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/config-guide/revert-split-database.html
- https://devdocs.magento.com/guides/v2.3/config-guide/revert-split-database.html

whatsnew
Added a new topic with instructions for[ reverting from a split database to a single database ](https://devdocs.magento.com/guides/v2.4/config-guide/revert-split-database.html)implementation

### CODE REVIEW

File: src/guides/v2.3/config-guide/revert-split-database.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - Setup
+---
+
+The [split database]({{ page.baseurl }}/config-guide/multi-master/multi-master.html) feature was deprecated in version 2.4.2 of {{ site.data.var.ee }} and {{ site.data.var.ce }}. Follow these instructions to revert from a split database to a single database implementation.
+
+## Revert a split database implementation
+
+Reverting from a split database to a single database implementation involves creating backups of the `magento_quote` and `magento_sales` databases before loading them into the single `magento_main` database.
+
+In this example, all three databases are installed on the same host named `magento2-mysql`.
```
Comment (paliarush) [inline src/guides/v2.3/config-guide/revert-split-database.md:17]: It is worth mentioning that in the examples below "root" MySQL user is used, but it has to be replaced with the appropriate user name for the Magento instance.

**Review by paliarush — APPROVED** [review]

File: src/guides/v2.3/config-guide/revert-split-database.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+group: configuration-guide
+title: Configure a single database from a split database
+ee_only: true
+functional_areas:
+  - Configuration
+  - System
+  - Setup
+---
+
+The [split database]({{ page.baseurl }}/config-guide/multi-master/multi-master.html) feature was deprecated in version 2.4.2 of {{ site.data.var.ee }} and {{ site.data.var.ce }}. Follow these instructions to revert from a split database to a single database implementation.
```
Comment (keharper) [inline src/guides/v2.3/config-guide/revert-split-database.md:11]: Your headers say this is an EE feature. Fix the discrepancy, and if it's available on EE only, add the `edition: ee-only` header to the TOC.

File: src/guides/v2.3/config-guide/revert-split-database.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+group: configuration-guide
+title: Configure a single database from a split database
```
Comment (keharper) [inline src/guides/v2.3/config-guide/revert-split-database.md:3]: I think "Migrate" or "Revert" would be a better verb. Whatever you decide, the file name should better align with the title.

File: src/guides/v2.3/config-guide/revert-split-database.md (hunk 1 above)
Comment (jeff-matthews) [reply src/guides/v2.3/config-guide/revert-split-database.md]: Thanks @paliarush. Done.

**Comment by jeff-matthews** [comment]
running tests

File: src/guides/v2.3/config-guide/revert-split-database.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+   mysql -h "magento2-mysql" -u root -p -e "DROP DATABASE magento_sales;"
+   ```
+
+1. Drop the `magento_quote` database:
+
+   ```bash
+   mysql -h "magento2-mysql" -u root -p -e "DROP DATABASE magento_quote;"
+   ```
+
+1. Remove the deployment configuration for `checkout` and `sales` in the `connections` section of the `env.php` file.
```
Comment (keharper) [inline src/guides/v2.3/config-guide/revert-split-database.md:55]: Can this step and the next be combined?

File: src/guides/v2.3/config-guide/revert-split-database.md (hunk 2 above)
Comment (jeff-matthews) [reply src/guides/v2.3/config-guide/revert-split-database.md:11]: The correct metadata tag is `ee_only: true` and already exists on line 4.

File: src/guides/v2.3/config-guide/revert-split-database.md (hunk 4 above)
Comment (jeff-matthews) [reply src/guides/v2.3/config-guide/revert-split-database.md:55]: sure

**Review by keharper — APPROVED** [review]

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**